crates/dojo-core/src/world.cairo

```diff
@@ -202,12 +221,25 @@ mod world {
         values: Span<felt252>,
     }
 
+    #[derive(Drop, starknet::Event)]
+    struct StoreSetEntity {
```

Comment:
After checking with @Larkooo and @lambda-0x, let's change that to `StoreUpdateRecord`. Like so, we can also have the `.update` instead of `.set` on the `Value` model. 👍